### PR TITLE
Seed Phrasing ✌🏽

### PR DESCRIPTION
**Pull request by adrianbona**
A little refactoring before generating common modules.

### CODE REVIEW

File: app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    React.useEffect(() => {
+        if (seedPhraseIsCorrect) {
+            setVerifyPhraseButtonText("Verify Phrase")
+        } else {
+            setVerifyPhraseButtonText("Phrase isn't correct yet!")
+        }
+    }, [seedPhraseIsCorrect]);
+
+    React.useEffect(() => {
+        setSeedPhraseIsCorrect(chosenPhrase.length === 12);
```
Comment (ACatThatPrograms) [inline app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx:30]: Good pluggable code, can easily update to correct verification functions when written :star:

File: app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    }, [chosenPhrase]);
+
+    const handlePhraseClick = seed => {
```
Comment (ACatThatPrograms) [inline app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx:33]: :book: Lingo/readability thing, if we can change `seed` to `word`. _Seed Phrase_ generally encompasses the entire mnemonic. Took me a second to process on a straight read that this was not the whole seed being passed, but rather one piece of it ( the clicked word ).

**Review by ACatThatPrograms — CHANGES_REQUESTED** [review]
This PR is :fire: and looks great.

Just the lingo thing and I am :smile_cat:

File: app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx (hunk 2 above)
Comment (adrianbona) [reply app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx:33]: I doubted for a while indeed on the variable's name. Let me fix it.

Comment (ACatThatPrograms) [reply app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx:33]: This occurs in a few other places. So anywhere seed is referring to a ~6 character string that is a piece of the mnemonic

**Review by ACatThatPrograms — APPROVED** [review]
Yes! Amazing, thank you! :fire:

**Pull request merged**